data/remnant missions.txt

```diff
@@ -1506,3 +1510,150 @@ mission "Remnant: Broken Jump Drive job"
+			`	As the <ship> levels off at the designated altitude, you strap yourself into the pressure suit. Stepping into the airlock, you lower the shields so you can exit onto the hull. Out on the surface, you admire the misty kaleidescope of colors filtering down through the cloud layers, playing over distant shapes moving in the mists. Before long several void sprites emerge, weaving and dancing among the clouds. They swirl for a few minutes around your cargo bay doors as if waiting for something then pulse briefly with flickers of light and then drop into the clouds below. The third void sprite flutters around the <ship>, poking at things and sliding over them. You quickly think to turn on your video recorder to catch the scene. The Void Sprite notices your brief movement, and turns towards you with a lunge. As it closes on you, your attention is drawn to what you suspect is its mouth: A zig-zagging crack that peels open like a zipper to reveal an opening that resembles an industrial grinder.`
+			`	Up close, the Void Sprite is massive: Most of its tentacles are easily as thick as your leg, and move with surprising speed and precision. It moves through the air like a strange kind of fish and you can make out a low vibration you suspect is probably from the air whistling over it as it closes with you. Facing down this rapidly approaching giant, you feel something click in the back of your mind and a wave of paralysis washes over you. In a moment of lucidity moments before the void sprite hits, you recall that your shields are keyed to your equipment: You could activate them to protect yourself from this attack, and they would extend around you. It probably wouldn't be too healthy for the Void Sprite, though.`
+			choice
+				`	(Activate shields)`
+				`	(Do not activate shields)`
+					goto holdground
+			`	Your mind latches onto the idea and you trigger the shields in a rush. The millisecond that follows feels like an eternity: The void sprite is just a few meters away, one tendril reaches out and touches you just as the shield slams into place. You see it strike the tendril and severs it cleanly in the process of re-establishing a sealed barrier. The same instant that the pressure on your mind with the paralysis turns off, as if it was simply a dream. You momentarily stare at the immobile void sprite, both frozen in shock over what just happened. Then time reasserts itself and sprite spasms away from the ship with an unearthly crescendo of tones. It plunges down into the clouds, leaving you standing on your hull with the floating severed piece of tendril.`
```

Comment:
Just going to say that shields are meant to be very close to your ship, I wouldn't have though that you could: 
A: extend it in any way or
B: have it cut off about 1m of anything.

Reply:
The lore doesn't seem too clear on that - and ships have a lot of greebles to warp around. I don't see too much difference between a turret that moves around and a heavy duty pressure suit that is magnetically clamped to the hull. (The robots from a System's core aren't affected, either.)

Based on a rather loose interpretation of the generators vs re-generators descriptions, I've generally taken the view that primitive shields and those produced by generators tended to be spherical (not perfect spheres, probably egg shaped in most cases) as that is the easiest, whereas re-generators and more advanced shielding tended to hug the ship more, which costs more energy but is more flexible.

But that is something that should be discussed. If something is projecting through where the shield will be, what happens to it? Could a piercer be used to weaken a shield, something be inserted through, and then used thereafter as a conduit for getting stuff into or out of a shielded area? Would it simply be sheared off? Or would it be eroded over time, kind of like a laser burning its way through a blast door?

Reply:
`same instant that the` -> `same instant, the`

Reply:
Thanks, fixed!